docs/reference/commandline/build.md

```diff
@@ -345,6 +345,13 @@ This will use a file called `Dockerfile.debug` for the build instructions
 instead of `Dockerfile`.
 
 ```bash
+$ docker build -f - . < ../Dockerfile
```

Comment:
Yes, that `-f` example would fail, point was that it may not be clear at a glance from `<` that it's `stdin`. Perhaps `cat <foo> | docker build -f - .`? 

Open to better suggestions though, not sure what's best 😅